src/material/schematics/ng-add/theming/create-custom-theme.ts

```diff
@@ -12,38 +12,38 @@ export function createCustomTheme(name: string = 'app') {
 // Custom Theming for Angular Material
 // For more information: https://material.angular.io/guide/theming
 @use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
```

Comment:
We probably don't want to include experimental stuff. Should we just wait on this until its moved out of experimental?

Reply:
I wanted to finish these instead of waiting for the code to be moved out of experimental. Once we do move them, it'll be quick to change the schematic.